### DISCUSSION TITLE
Using Dynamic Select with existing result file

### DISCUSSION THREAD

**Discussion by Arinomo** [OpenModelica/OpenModelica#9630] [Q&A] [answered]
Like the title state, is it possible to "replay" simulation result with dynamic select using an existed result file?

 I'm using  OMv1.19.0-dev-590-g0897872ac1 and noticed I can not play an already existed result file. I seem can not set the result to "active" and i assume that is the problem=

Best Regards

**Comment by adeas31** [comment, marked as answer]
No, it is not possible yet. However, it is doable so feel free to make a ticket about it.

**Reply by Arinomo** [reply]
OK, will do. thanks.
